examples/core/cross-blueprint-call/README.md

```diff
@@ -9,15 +9,15 @@ In this example, we demonstrate two ways of calling a function or method defined
 If the function or method is from an already published package, we need to
 1. Export the ABI of the blueprint using tools like `resim`
     ```
-    resim export-abi <PACKAGE_ADDRESS> Airdrop
+    resim export-abi <PACKAGE_REF> Airdrop
     ```
 2. Import the ABI into our package, e.g.,
     ```rust
     import! {
     r#"
     {
-        "package": "01bda8686d6c2fa45dce04fac71a09b54efbc8028c23aac74bc00e",
-        "name": "Airdrop",
+        "package_ref": "01bda8686d6c2fa45dce04fac71a09b54efbc8028c23aac74bc00e",
```

Comment:
hrm, feel like this should also just be package, package_ref feels weird, as its not a reference at this point but an actual value.

Reply:
This right hand side is a package address, not a package.

Now I view `XxxxRef` in two ways:
- It's a reference to the substate object, so I can invoke it.
- It's an identifier of the substate object, so I can pass it around.

Another approach is to need two types for the two use cases, like before we have `Component` and `Address`.

Reply:
Or some types like `PackageId` (identifer) and `PackageRef` (reference)

Reply:
To make it even worse is mutable semantics. Ideally, there should be `PackageRef` and `PackageMutRef`, unless we have `Package`, `&Package` and `&mut Package`.